lib/logger/lib/logger/utils.ex

```diff
@@ -153,7 +153,7 @@ defmodule Logger.Utils do
   ## encoding
 
   defp collect_cc(:encoding, [?l | t], args, used_format, used_args, opts),
-    do: collect_cc(:done, t, args, [?l | used_format], used_args, %{opts | char_lists: false})
+    do: collect_cc(:done, t, args, [?l | used_format], used_args, %{opts | charlists: false})
```

Comment:
You are right, `:as_lists` should be it. I think `false` works by accident, a test should actually tell us if `:as_lists` is correct or not.